tests/extensions/test_check_raise_docs.py

```diff
@@ -105,6 +105,63 @@ def my_func(self):
         ):
             self.checker.visit_raise(raise_node)
 
+    def test_ignores_exception_inheritance_builtin(self) -> None:
+        node = astroid.extract_node(
+            '''
+        def my_func(self):
+            """This is a docstring.
+
+            Raises:
+                Exception: Never
+            """
+            raise ValueError('hi')
+        '''
+        )
+        raise_node = node.body[0]
+        with self.assertNoMessages():
+            self.checker.visit_raise(raise_node)
+
+    def test_ignores_exception_inheritance_custom(self) -> None:
+        node = astroid.extract_node(
+            '''
+        class CustomError(NameError):
+            pass
+
+        def my_func(self):
+            """This is a docstring.
+
+            Raises:
+                NameError: Never
+            """
+            raise CustomError('hi')
+        '''
+        )
+        raise_node = node.body[0]
+        with self.assertNoMessages():
+            self.checker.visit_raise(raise_node)
```

Comment:
How about adding a test that's the opposite of this? If `CustomError` is in the docstring and we raise `NameError`, we emit should emit `missing-raises-doc` with `NameError`.

This test would ensure we don't get `expected` and `found` mixed up and we're checking against the right chain of ancestors.

Reply:
fair point. Added